jekyll/_cci2/notifications.md

```diff
@@ -67,20 +52,17 @@ Email notifications will look like the following:
 
 ## Enable Web Notifications
 
-1. Go to your [CircleCI user settings](https://circleci.com/account/notifications){:rel="nofollow"}. The link to turn on permissions is at the bottom in the Web Notifications section as shown in the screenshot:
-![](  {{ site.baseurl }}/assets/img/docs/notification-default-message.png)
+Perform the following to enable web notifications:
 
-2. Click the link to turn on permissions. Your browser will ask you to confirm that you want to allow notifications from CircleCI.
+1. Go to your [CircleCI user settings](https://circleci.com/account/notifications){:rel="nofollow"}. Enable the toggle for "Web Notifications" at the bottom of the document.
 
-3. Click Allow as shown in the screenshot example.
-![](  {{ site.baseurl }}/assets/img/docs/notification-allowing.png)
+2. Your browser will ask you to confirm that you want to allow notifications. Click `Allow`. See the screenshot below for additional details:
 
-4. Select the radio button to show notifications for completed builds. **Note:** To select the radio button, you may need to reload the page.
+![]({{ site.baseurl }}/assets/img/docs/notifications-enable-web.png)
 
 If you've previously denied CircleCI permission to send you web notifications
-then you'll need to turn those permissions on in your browser, as CircleCI can't
-request permission in this case. Use your browser settings to control notifications. In Google Chrome you can do this by clicking the lock icon in the URL bar and selecting Notifications from the Permissions Settings as shown in the following animation:
-![](  {{ site.baseurl }}/assets/img/docs/notification-granting.gif)
+you will need to manually turn those permissions on in your browser, as CircleCI can't
```

Comment:
Replace "can't" with "cannot." Generally speaking, you should try to avoid contractions whenever possible.